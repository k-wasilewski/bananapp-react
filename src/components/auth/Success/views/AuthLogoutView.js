import {Redirect} from 'react-router-dom';
import React, {Component} from 'react';

class AuthLogoutView extends Component {
    render() {
        return (
            <Redirect to={{
                pathname: '/',
                state: { logout: true }
            }}/>
        );
    }
}
export default AuthLogoutView;
