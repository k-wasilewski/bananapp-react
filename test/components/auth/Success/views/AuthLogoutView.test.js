import React from 'react';
import renderer from 'react-test-renderer';
import store from "../../../../../src/redux/store";
import {BrowserRouter} from "react-router-dom";
import AuthLogoutView from "../../../../../src/components/auth/Success/views/AuthLogoutView";
import {Provider} from "react-redux";

describe("AuthLogoutView rendering specification", () => {
    it('AuthLogoutView is rendered', () => {
        const component = renderer.create(
            <Provider store={store}>
                <BrowserRouter>
                    <AuthLogoutView />
                </BrowserRouter>
            </Provider>
        );
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});