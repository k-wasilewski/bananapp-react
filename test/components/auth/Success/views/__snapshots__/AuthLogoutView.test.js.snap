// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthLogoutView rendering specification AuthLogoutView is rendered 1`] = `null`;
